elasticsearch-cluster-CCR/test/README.md

```diff
@@ -0,0 +1,57 @@
+# Run CCR (Cross Cluster Replication) test on Elasticsearch
+
+To perform the tests, two variables are necessary in the execution of the docker image, they are:</br>
+
+ - **LEADER_CLUSTER_IP**
+ - **FOLLOWER_CLUSTER_IP**
+
+ The variable **LEADER_CLUSTER_IP** must be filled in with Leader Elasticsearch Cluster access without port 9200, for example:</br>
+
+- If my Leader Elasticsearch cluster is acessible by `http://11.222.333.444:9200`, the environment variable must be filled with `http://11.222.333.444`</br>
+
+```bash
+LEADER_CLUSTER_IP=http://11.222.333.444
+```
+
+The variable **FOLLOWER_CLUSTER_IP** must be filled in with Follower Elasticsearch Cluster access without port 9200, for example:</br>
+
+- If my Follower Elasticsearch cluster is acessible by `http://22.333.444.555:9200`, the environment variable must be filled with `http://22.333.444.555`</br>
+
+```bash
+FOLLOWER_CLUSTER_IP=http://22.333.444.555
+```
+---
+## Build docker image
+Run docker build to build the test image.</br>
+
+```bash
+docker build . -t elasticsearch_test:1.0 
+```
+
+## Run docker image
+Run elasticsearch test image.</br>
+
+```bash
+docker run -e LEADER_CLUSTER_IP=http://11.222.333.444 -e FOLLOWER_CLUSTER_IP=http://22.333.444.555 elasticsearch_test:1.0 
+```
+
+## Test steps
```

Comment:
where are the commands needed to run the tests written in JS?
please specify in a step by step list, the declaration of the environment variables, npm install, npm run, and so forth